ccpp/data/GFS_typedefs.F90

```diff
@@ -6027,7 +6032,11 @@ subroutine tbd_create (Tbd, IM, Model)
 
     if (Model%imfdeepcnv == Model%imfdeepcnv_gf) then
        allocate(Tbd%cactiv(IM))
+       allocate(Tbd%cactiv_m(IM))
+       allocate(Tbd%aod_gf(IM))
```

Comment:
@climbfuji @hannahcbarnes I see that this is only allocated for GF here, which strengthens the argument to qualify the standard name.